spec/std/isa/csr/seed.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) Anirudh Narang
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+
+# yaml-language-server: $schema=../../schemas/csr_schema.json
+
+$schema: "csr_schema.json#"
+kind: csr
+name: seed
+long_name: Entropy Seed Register
+address: 0x015
+priv_mode: U
+length: 64
+writable: false
+definedBy:
+  name: Zkr
+  version: ">=1.0.0"
+description: |
+  The `seed` CSR exposes physical entropy directly to software. This register is useful for cryptographic
+  applications and random number generation. Reads may have side effects depending on the implementation.
+
+sw_read(): |
+  return read_entropy();
+
+fields:
```

Comment:
The fields need to match the definition in the spec:
- OPST: 31:30
- custom: 23:16
- entropy: 15:0

read/"write" access is managed through `mseccfg.sseed` and `mseccfg.useed`, depending on mode. These restrictions need to be realized in `sw_read()/sw_write()` methods.